nginx.montagu.conf

```diff
@@ -49,6 +49,23 @@ server {
     location /reports/ {
         proxy_pass http://report/;
     }
+
+    location /shiny/ {
+        rewrite ^/shiny/(.*)$ /$1 break;
+        
+        proxy_pass http://shiny_proxy;
+        proxy_redirect http://shiny_proxy/ $scheme://$host/shiny/;
+        proxy_http_version 1.1;
+        proxy_set_header Upgrade $http_upgrade;
+        proxy_set_header Connection $connection_upgrade;
```

Comment:
I guess the bit above that I didn't understand relates to this... but I don't know what the Upgrade and Connection headers do.